### DISCUSSION TITLE
Markdown question

### DISCUSSION THREAD

**Discussion by abunavas** [community/community#23237] [New to GitHub]
<p>I am not sure if I can ask here, please tell me is this is wrong forum.</p>
<p>I have a long list in markdown like this:</p>
<pre><code class="lang-auto">* Lorem
* ipsum
* dolor
* sit
* amet
</code></pre>
<p>I want to get rid of *, but when I try to replace it wit two spaces or <code>&lt;br&gt;</code>, in html file I receive one long paragraph. So my question is how to replace * with Enter? I hope to have lines like this:</p>
<p>Lorem<br>
ipsum<br>
dolor<br>
sit<br>
amet</p>
<p>Thank you.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Try this:</p>
<pre><code class="lang-markdown">Lorem\
ipsum\
dolor\
sit\
amet
</code></pre>
<p>I’m not sure if it works with every Markdown parser, but it works on Github and with Pandoc. :slightly_smiling_face:</p>

**Comment by abunavas** [comment]
<aside class="quote no-group" data-username="airtower-luna" data-post="2" data-topic="152765">
<div class="title">
<div class="quote-controls"></div>
 airtower-luna:</div>
<blockquote>
<p>I’m not sure if it works with every Markdown parser, but it works on Github and with Pandoc.</p>
</blockquote>
</aside>
<p>Thank you. I know this, but <a href="https://www.markdownguide.org/basic-syntax/" class="inline-onebox" rel="noopener nofollow ugc">Basic Syntax | Markdown Guide</a> say it is a bad practice</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/d/ed1c5ba10ef324667c79aa13a5d8fae408e8f163.png" data-download-href="/uploads/short-url/xPzURefF0R41oVCOkkSz3jtV5Pd.png?dl=1" title="Screenshot 2020-12-30 203451" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/11981078/181075692-80501590-5ade-4dd5-9ba2-9d324eef28d4.png" alt="Screenshot 2020-12-30 203451" data-base62-sha1="xPzURefF0R41oVCOkkSz3jtV5Pd" width="690" height="203" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/d/ed1c5ba10ef324667c79aa13a5d8fae408e8f163_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-12-30 203451</span><span class="informations">874×258 4.71 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>EDIT: Well, I’ll use it…</p>
